Presentation.Web/Tests/01_Parallel/it-system/Usage/Tabs/GDPR.System.e2e.spec.ts

```diff
@@ -96,11 +95,11 @@ describe("Global admin is able to", () => {
     function fillOutDateFields() {
         console.log("Entering a date into date fields");
         return ItSystemUsageGDPRPage.getRiskAssesmentDateField().sendKeys(dateValue)
+            .then(() => ItSystemUsageGDPRPage.getPlannedRiskAssessmentDateField().sendKeys(dateValue))
```

Comment:
don't use the same datavalue for two fields... use another one and name them accordingly